ci/run_style_check

```diff
@@ -10,8 +10,12 @@ files=(
   spelling.txt
 )
 
+if [ "$KVER" != "master" ]; then
```

Comment:
KVER is undefined, that cannot work,

Reply:
Thanks for the feedback! I forgot that `env` in the `include` overrides the global `env`.